packages/v4/patternfly-docs/patternfly-docs.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   hasFooter: true,
   hasVersionSwitcher: true,
   hasDesignGuidelines: true,
-  hasDarkThemeSwitcher: false,
+  hasDarkThemeSwitcher: true, // change back before committing. DONT LET ME COMMIT THIS!
```

Comment:
You committed this - do you mean don't merge this?

Reply:
Oops, yep! When we get an approval on the PR from @mceledonia or @mmenestr, I'll remove that.